src/java/org/apache/cassandra/transport/CBUtil.java

```diff
@@ -46,7 +46,7 @@
 import org.apache.cassandra.utils.Pair;
 import org.apache.cassandra.utils.TimeUUID;
 import org.apache.cassandra.utils.UUIDGen;
-
+import org.apache.cassandra.utils.memory.MemoryUtil;
```

Comment:
should not be there some free line after imports and before the beginning of comments? This feels strange.

Reply:
@NateAdere  why was this comment resolved?